internal/archive/archive.go

```diff
@@ -184,6 +238,13 @@ func openUbuntu(options *Options) (Archive, error) {
 					return nil, err
 				}
 				release = index.release
+				if !index.supportsArch(options.Arch) {
+					// Release does not support the specified architecture, do
+					// not add any of its indexes.
+					logf("Warning: ignoring %s %s %s suite (unsupported arch %s)...",
```

Comment:
Similar comment as earlier on.

Also, do we need to report?  Isn't this going to be weird as we'll constantly emitting a warning for a situation where there's nothing to be fixed, as long as that architecture is in use?

Reply:
Agreed offline to omit this warning because there is nothing the user can do, to know whether we are using pro archives we can rely on the log messages that already specify pro.